main.java

```diff
@@ -0,0 +1,9 @@
+class Main {
+    public static void main(String[] args) {
+        Machine[] machines = {new Truck(), new Buldozer(), new Excavator()};

```

Comment:
There is a typo in the class name `Buldozer`. It should be `Bulldozer` as per the task description .